x-pack/plugins/apm/public/components/app/correlations/context_popover/top_values.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { Fragment } from 'react';
+import {
+  EuiButtonIcon,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiProgress,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { IndexPatternField } from '../../../../../../../../src/plugins/data/common';
+import { TopValuesStats } from '../../../../../common/search_strategies/field_stats_types';
+import { asPercent } from '../../../../../common/utils/formatters';
+import { useTheme } from '../../../../hooks/use_theme';
+
+interface Props {
+  stats: TopValuesStats;
+  barColor?: 'primary' | 'secondary' | 'danger' | 'subdued' | 'accent';
+  compressed?: boolean;
+  onAddFilter?: (
+    field: IndexPatternField | string,
+    value: string,
+    type: '+' | '-'
+  ) => void;
+}
+
+export function TopValues({ stats, barColor, onAddFilter }: Props) {
+  const {
+    topValues,
+    topValuesSampleSize,
+    topValuesSamplerShardSize,
+    count,
+    isTopValuesSampled,
+    fieldName,
+  } = stats;
+  const theme = useTheme();
+
+  const progressBarMax = topValuesSampleSize ?? count;
+  return (
+    <div
+      data-test-subj="apmCorrelationsContextPopoverTopValues"
+      style={{
+        minWidth: `calc(${theme.eui.euiSizeXXL} * 6.5)`,
+        maxWidth: `calc(${theme.eui.euiSizeXXL} * 7.5)`,
+      }}
+    >
+      {Array.isArray(topValues) &&
+        topValues.map((value) => (
+          <EuiFlexGroup gutterSize="xs" alignItems="center" key={value.key}>
```

Comment:
The popover in Discover adds an `EuiSpacer size=s` in between each of the values, and I prefer the extra spacing between the rows that that provides:

Discover:
![image](https://user-images.githubusercontent.com/7405507/136030973-da342a89-4537-4a84-8af6-9625c076a61f.png)

Compared to:
![image](https://user-images.githubusercontent.com/7405507/136031036-bb4adcb1-931a-44d1-983c-685e0f6fbaf1.png)

Reply:
@peteharverson @qn895 agreed! more spacing helps

Reply:
Updated in https://github.com/elastic/kibana/pull/113679/commits/c333b1a40549258941d24d2e81ca0b4d3d5cbc79. Definitely looks a lot better! 
<img width="374" alt="Screen Shot 2021-10-05 at 11 54 56" src="https://user-images.githubusercontent.com/43350163/136068834-6852d927-256f-4365-91f0-1828bce11897.png">